node/node.go

```diff
@@ -236,8 +232,9 @@ func (n *Node) GetGenesis() *tmtypes.GenesisDoc {
 }
 
 // GetGenesisChunks returns chunked version of genesis.
-func (n *Node) GetGenesisChunks() []string {
-	return n.genChunks
+func (n *Node) GetGenesisChunks() ([]string, error) {
+	err := n.initGenesisChunks()
+	return n.genChunks, err
```

Comment:
Ok, @tzdybal prefers the other way - which way should I take?

Reply:
Or did I misunderstand something about how the error should be handled 😅

Reply:
Oh I meant instead of returning `return n.genChunks, err`, return `return n.genChunks, nil` and do an explicit

```go
if err != nil {
    return nil, err
}
```

before that.